src/robot/conf/languages.py

```diff
@@ -202,12 +271,156 @@ class Fi(Language):
     test_timeout = 'Testin Aikaraja'
     task_timeout = 'Tehtävän Aikaraja'
     test_tags = 'Testin Tagit'
+    task_setup = 'Tehtävän Alustus'
+    task_teardown = 'Tehtävän Purku'
+    task_template = 'Tehtävän Malli'
+    task_timeout = 'Tehtävän Aikaraja'
```

Comment:
- Task settings are here now twice.
- Setup/teardown have conflicting terms in different places. Setup is using both `Alustus` and `Ylösajo` and teardown uses `Purku` and `Alasajo`. I don't have a super strong opinion on which to use, but the usage should be consistent. I think it's now best to use `Alustus` and `Purku` as that's what tests use. Could then be changed later if needed.

Reply:
Fixed. I will need to reflect it in Crowdin to have it all in one place